### PR TITLE
Fix e2e errors

### PR DESCRIPTION
**Pull request by flacatus**
Signed-off-by: flacatus <flacatus@redhat.com>

### What does this PR do?
Add 6 minutes to wait for pods to be ready and improve errors.

### CODE REVIEW

File: test/e2e/cmd/workspaces_test.go [hunk 1]
```diff
@@ -56,19 +56,19 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 	//TODO: Have better improvement of errors.
 	if err := controller.CreateAllOperatorRoles(); err != nil {
-		_ = fmt.Errorf("Failed to create roles in clusters %s", err)
+		panic("Failed to create roles in clusters")
```
Comment (sleshchenko) [inline test/e2e/cmd/workspaces_test.go:59]: please add error message as well here and in panic below

File: test/e2e/cmd/workspaces_test.go [hunk 2]
```diff
@@ -56,19 +56,19 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 	//TODO: Have better improvement of errors.
```
Comment (sleshchenko) [inline test/e2e/cmd/workspaces_test.go:57]: Does this PR do improvement that this TODO means?

Comment (flacatus) [reply test/e2e/cmd/workspaces_test.go:57]: yes, I'll delete the comment.

**Pull request merged**